src/utils/buffer.ts

```diff
@@ -81,7 +80,10 @@ export const unpackToInt = (n: number, signed: boolean, buff: Buffer): [number,
     }
   } else if (n === 8) {
     // We have only signed ints here
-    if (parseInt(nodeMajorVersion) > 8) {
+    const nodeMajorVersion = get(process, 'versions.node');
+    // Browsers don't have access to versions.node and they don't have readBigInt64BE method
+    // slicedBuff.readIntBE(0, 8) worked fine testing on electron and react native (desktop and mobile wallets)
+    if (nodeMajorVersion && parseInt(nodeMajorVersion.split('.')[0]) > 8) {
```

Comment:
Also here you can check only `parseInt(nodeMajorVersion) > 8` because:
```
parseInt('12.34.32') > 8
true
parseInt(undefined) > 8
false
parseInt(null) > 8
false
parseInt('randomstr') > 8
false
parseInt('12.32.beta-rc') > 8
true
```

Reply:
This code has changed, can you see again please?